user-guide/Reference/Metrics/alarm_benchmarks.md

```diff
@@ -50,6 +50,15 @@ Client:
 - 16GB RAM
 - Windows 10 Enterprise
 
+### Metric 16 & 17
+
+Client:
+
+- Intel(R) Core(TM) i7-9700 CPU @ 3.00GHz   3.00 GHz
```

Comment:
Is this repetition of "3.00 GHz" OK, or is this a mistake?

Reply:
It was a mistake. It is corrected now.